tests/unit/sproxyd.js

```diff
@@ -68,6 +68,10 @@ function makeResponse(res, code, message, data, md) {
     res.end();
 }
 
+function consumeAndMakeResponse(req, res, code, message, data, md) {
+    req.resume().on('end', () => makeResponse(res, code, message, data, md));
```

Comment:
not sure we want to consume all the time : a PUT may have a very large data, we may not want to consume (stream!) everything on error ?
→ in that case, we don't want to arbitrary introduce that behavior in tests, as we may make the code depend in that assertion...
→ is the 'dropping' of the data not already handled by the socket reuse? my understanding is that "socket reuse" is lower level, but should behave just like having a separate socket...